frontend/src/components/duck/operations.js

```
@@ -0,0 +1,410 @@
+import actions from './actions'
+import {checkQuery} from '../common.js'
```

ESLint

[error] 'checkQuery' is defined but never used.

(no-unused-vars)